### PR TITLE
Added 2 game sites

### PR DESCRIPTION
**Pull request by K2SOsint**
forums.thesims.com
discussions.ubisoft.com

Tested & working.

### CODE REVIEW

File: web_accounts_list.json [hunk 1]
```diff
@@ -4711,6 +4711,28 @@
...
+           "account_missing_code" : "404",
+           "known_accounts" : ["babykittyjade", "simgurumorgan"],
+           "category" : "gaming",
+           "valid" : true
+          },
+          {
+           "name" : "ubisoft.com",
+           "check_uri" : "https://discussions.ubisoft.com/user/{account}",
+           "account_existence_code" : "200",
+           "account_existence_string" : "Ubisoft Discussion Forums",
```
Comment (WebBreacher) [inline web_accounts_list.json:4730]: This is found on the pages with no accounts. Consider changing it to `account-list-navigation`

File: web_accounts_list.json [hunk 2]
```diff
@@ -4711,6 +4711,28 @@
...
+           "account_missing_string" : "Not Found",
```
Comment (WebBreacher) [inline web_accounts_list.json:4731]: Let's be more precise here as you could have profiles with the words `Not Found` in them. Consider changing to `<title>Not Found | Ubisoft`

**Review by WebBreacher — CHANGES_REQUESTED** [review]
Please make these changes and rebase you JSON file as it is out of sync with the repo due to other submissions.

File: web_accounts_list.json (hunk 2 above)
Comment (K2SOsint) [reply web_accounts_list.json:4731]: Will do.

File: web_accounts_list.json (hunk 1 above)
Comment (K2SOsint) [reply web_accounts_list.json:4730]: Found an <H2 class that is not on the 'Not found' page, will use that as existence string instead.

**Pull request closed without merging**